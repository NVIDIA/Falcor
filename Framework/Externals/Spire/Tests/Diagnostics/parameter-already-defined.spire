// re-use parameter name

int foo( int a, float a ) { return 0; }
