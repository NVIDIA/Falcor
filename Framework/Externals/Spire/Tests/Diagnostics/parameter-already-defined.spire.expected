result code = -1
standard error = {
Tests/Diagnostics/parameter-already-defined.spire(3): error 30002: parameter 'a' already defined.
}
standard output = {
}
